skosprovider_getty/utils.py

```diff
@@ -283,12 +289,27 @@ def uri_to_graph(uri, **kwargs):
     '''
     s = kwargs.get('session', requests.Session())
     graph = rdflib.Graph()
-    try:
-        res = s.get(uri)
-    except requests.ConnectionError as e:
-        raise ProviderUnavailableException("URI not available: %s" % uri)
+    res = do_get_request(uri, s)
     if res.status_code == 404:
         return False
     graph.parse(data=res.content)
     return graph
 
+
+def do_get_request(url, session=None, headers=None, params=None):
+    if not session:
+        session = requests.Session()
+    try:
+        res = session.get(url, headers=headers, params=params)
+    except ConnectionError:
+        raise ProviderUnavailableException("Request could not be executed \
+                due to connection issues- Request: %s" % (url,))
+    except Timeout: # pragma: no cover
+        raise ProviderUnavailableException("Request could not be executed \
+                due to timeout - Request: %s" % (url,))
+    if res.status_code >= 500:
+        raise ProviderUnavailableException("Request could not be executed \
```

Comment:
I think that either the params or the response body would be useful to log as well in this case. Otherwise it's hard to find what was wrong from just the logs.